splash/qtrender_lua.py

```diff
@@ -343,10 +344,26 @@ def error(error_info):
         ))
 
     @command(async=True)
-    def go(self, url, baseurl=None, headers=None):
+    def go(self, url, baseurl=None, headers=None, http_method="GET", body=None, formdata=None):
+
         if url is None:
             raise ScriptError("'url' is required for splash:go")
 
+        if http_method not in OPERATION_QT_CONSTANTS:
+            raise ScriptError("incorrect HTTP method: {}".format(http_method))
+
+        if formdata:
+            body = self.lua.lua2python(formdata)
+            if isinstance(body, dict):
+                body = urlencode(body)
+            else:
+                raise ScriptError("formdata argument for go() must be Lua table")
```

Comment:
> What if user passes {'foo', 'bar', 'baz'}? It is still a Lua table, is it accepted?

this will be converted to {1: "foo", 2:"bar", 3:"baz"} probably not what user will expect, but should not fail in splash I suppose. you think we should check for cases like this? Can be difficult to detect